src/mapGenerator/GreenlandGenerator.cpp

```diff
@@ -0,0 +1,415 @@
+// Copyright (c) 2005 - 2015 Settlers Freaks (sf-team at siedler25.org)
+//
+// This file is part of Return To The Roots.
+//
+// Return To The Roots is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 2 of the License, or
+// (at your option) any later version.
+//
+// Return To The Roots is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
+
+#include <vector>
+#include <cstdlib>
+#include <boost/range/algorithm.hpp>
+
+#include "mapGenerator/Defines.h"
+#include "mapGenerator/GreenlandGenerator.h"
+#include "mapGenerator/VertexUtility.h"
+#include "mapGenerator/ObjectGenerator.h"
+
+#ifndef MIN
+#define MIN(x, y) x < y ? x : y
```

Comment:
I used it before, then saw some error message about "min"/"max" not being in the std namespace. Just noticed that probably only an include was missing. I'll change it.